### PR TITLE
Proofreading ch05-03

### PR DESCRIPTION
**Pull request by Jimskapt**
I've proofread #116, @joeljpresent are you OK with this ?

### CODE REVIEW

File: FRENCH/src/ch05-03-method-syntax.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ## Method Syntax
 -->
 
-## La syntaxe de méthode
+## La syntaxe des méthodes
```
Comment (joeljpresent) [inline FRENCH/src/ch05-03-method-syntax.md:5]: What is wrong with `la syntaxe de méthode`?
I find this term better than `la syntaxe de la méthode` or `la syntaxe des méthodes` which sound like this is a syntax linked it one or several specific methods, whereas `la syntaxe de méthode` makes it clear that it simply refers a specific syntax.

File: FRENCH/src/ch05-03-method-syntax.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ instance of the struct the method is being called on.
 
 Les *méthodes* sont similaires aux fonctions : on les déclare avec le mot-clé
 `fn` et leur nom, elles peuvent avoir des paramètres et une valeur de retour, et
-elles contiennent du code qui est exécuté quand on les appelle depuis un autre
-endroit. Cependant, les méthodes diffèrent des fonctions en ce qu'elles sont
+elles contiennent du code qui est exécuté quand on les utilise depuis un autre
```
Comment (joeljpresent) [inline FRENCH/src/ch05-03-method-syntax.md]: Why use the imprecise verb `utiliser` instead of the more technical, well-suited verb `appeler`?

**Review by joeljpresent — COMMENTED** [review]
I am fine with most changes, except a few.

File: FRENCH/src/ch05-03-method-syntax.md (hunk 2 above)
Comment (Jimskapt) [reply FRENCH/src/ch05-03-method-syntax.md]: I've got some doubts on the conjugation of this verb, so I've changed the word, but I now realise it's the same problem, sorry :sweat:
So I will restore your word, `appelle`, very soon but I'm still not sure about the conjugation ... :confused:

File: FRENCH/src/ch05-03-method-syntax.md (hunk 1 above)
Comment (Jimskapt) [reply FRENCH/src/ch05-03-method-syntax.md:5]: I've found it much more natural this way (and sorry, but I'm still).

Don't you agree that we talk about a syntax that is applied on all methods ?

*(and at first i've changed to `la syntaxe de la méthode` before choose `la syntaxe des méthodes`)*

Comment (joeljpresent) [reply FRENCH/src/ch05-03-method-syntax.md:5]: Let's go with `la syntaxe des méthodes`.

Comment (Jimskapt) [reply FRENCH/src/ch05-03-method-syntax.md:5]: Awesome ! thank you, joel ! 👍

**Pull request merged**